### PR TITLE
Add large font size scaling in webviews

### LINKED ISSUE #2137: Control font scaling in web views
For our own text components we can control font scaling with the `maxFontSizeMultiplier`, but web views don't seem to respect larger system font sizes. Look into our options for controlling this in our app (is there a prop? maybe we can set a base CSS font size for the web view?), and if there aren't any, ask the web team if there are options for our own website, at least.

### PR DESCRIPTION
**Pull request by angielt**
Closes #2137

### CODE REVIEW

File: src/components/FullPageWebView/FullPageWebView.tsx [hunk 1]
```diff
@@ -196,11 +198,26 @@ const FullPageWebView = ( ) => {
     }, [navigation, params.loggedIn, params.title] )
   );
 
+  const fontScalingJS
+  = `
```
Comment (kueda) [inline src/components/FullPageWebView/FullPageWebView.tsx:202]: Please don't start a line with an equals. IDK why eslint allows this.

File: src/components/FullPageWebView/FullPageWebView.tsx [hunk 2]
```diff
@@ -196,11 +198,26 @@ const FullPageWebView = ( ) => {
...
+    if( ${isLargeFontScale} ){
+      document.body.style.fontSize = '40px';
```
Comment (kueda) [inline src/components/FullPageWebView/FullPageWebView.tsx:204]: Do you really want to hard-code the size like this? Did you try a percentage?

File: src/components/FullPageWebView/FullPageWebView.tsx [hunk 3]
```diff
@@ -196,11 +198,26 @@ const FullPageWebView = ( ) => {
...
+      let paragraphs = document.querySelectorAll('p');
+      paragraphs.forEach(p => {
+        p.style.fontSize = '50px';
+      })
```
Comment (kueda) [inline src/components/FullPageWebView/FullPageWebView.tsx:208]: Why is modifying the P elements necessary if you're setting a base size for the BODY?

File: src/components/FullPageWebView/FullPageWebView.tsx [hunk 4]
```diff
@@ -196,11 +198,26 @@ const FullPageWebView = ( ) => {
...
+    }
+    true; // note: this is required, or you'll sometimes get silent failures
```
Comment (kueda) [inline src/components/FullPageWebView/FullPageWebView.tsx:210]: What kind of errors?

File: src/components/FullPageWebView/FullPageWebView.tsx (hunk 2 above)
Comment (angielt) [reply src/components/FullPageWebView/FullPageWebView.tsx:204]: I'll try percentages

File: src/components/FullPageWebView/FullPageWebView.tsx (hunk 3 above)
Comment (angielt) [reply src/components/FullPageWebView/FullPageWebView.tsx:208]: Not 100% sure why. Modifying the BODY only seems to affect the headings. My guess is that there might be some css that defines the font size specifically for P tags that overrides the BODY.

File: src/components/FullPageWebView/FullPageWebView.tsx (hunk 4 above)
Comment (angielt) [reply src/components/FullPageWebView/FullPageWebView.tsx]: Haven't experienced any errors personally but it was listed in the docs to make sure the string evaluates to a valid type

File: src/components/FullPageWebView/FullPageWebView.tsx (hunk 3 above)
Comment (kueda) [reply src/components/FullPageWebView/FullPageWebView.tsx:207]: That sounds like a web page hard-coding a font size... which is kind of out of our control, unless it's our own web page. Where were you seeing this?

**Review by kueda — APPROVED** [review]
I don't think this looks quite right in the Settings webview, but I do think it's an improvement, so let's merge.

**Pull request merged**